src/index.js

```diff
@@ -41,11 +41,33 @@ ReactGA.initialize('UA-532159-1');
 
 const history = createBrowserHistory();
 
+const parseUrl = (partialUrl) => {
```

Comment:
@cbernat, maybe you can use something like: [url-parse](https://www.npmjs.com/package/url-parse), what do you think?

Reply:
Good catch @leoslopez.

@cbernat we are already using [query-string library](https://www.npmjs.com/package/query-string) to parse URLs. For example https://github.com/FromDoppler/doppler-webapp/blob/a74e292ec583f733f650b7563677334d140929f5/src/components/Login/Login.js#L34

Reply:
@andresmoschini this library you've mention parses urls relaying in location base parsing, does not detect hash as part of a query string for example. 
 If you do ```queryString.parseUrl('https://foo.bar?foo=bar#hash'); ```
//=> {url: 'https://foo.bar', query: {foo: 'bar'}} --> ignoring hash
![image](https://user-images.githubusercontent.com/2439363/61479916-38242b00-a96b-11e9-9445-3c07f1e66e0b.png)

I think I will go with Leo's suggestion.

Reply:
We will chat about it: https://makingsense.slack.com/archives/CGJ72QFQX/p1563477192179800